### PR TITLE
[16.0][ADD] pos_partner_birthdate module

### PR DESCRIPTION
**Pull request by ecino**
Module for adding the birthdate in the customer screen of the POS.

### CODE REVIEW

File: pos_partner_birthdate/__init__.py
Comment (legalsylvain) [inline pos_partner_birthdate/__init__.py]: Nitpicking : licence are useless for empty init file. (x2)

File: pos_partner_birthdate/models/pos_session.py
Comment (legalsylvain) [inline pos_partner_birthdate/models/pos_session.py]: https://github.com/OCA/pos/blob/16.0/pos_customer_comment/models/pos_session.py#L11

Looks more concise.

**Review by legalsylvain — APPROVED** [review]
Thanks !

Minor comment inline. Otherwise lgtm. (Code review / no test)

**Comment by ecino** [comment]
Thank you for the review! I made the suggested corrections.

**Review by hitrosol — APPROVED** [review]
LGTM

**Comment by ecino** [comment]
@legalsylvain Can this command be done by anyone or not?

**Comment by legalsylvain** [comment]
Hi, 
- the bot gives full access to the [OCA/pos maintainers team, see members](https://github.com/orgs/OCA/teams/pos-maintainers/members)
- Also if you are marked as maintainers of module(s), you can merge patch on your modules.

The rule is to wait for 2 positive reviews and 5 days before merging.

**Comment by legalsylvain** [comment]
3) You can not merge your own PR that brings the module. only patches in futur PRs, if the module is accepted.

4) Merging take time as the CI has to be executed. No need to call again the bot in the meantime. ;-)

**Pull request merged**
